### PR TITLE
P152778506 feature doc compute totals

### PR DESCRIPTION
**Pull request by hschilling**

### CODE REVIEW

File: openmdao/core/tests/test_problem.py [hunk 1]
```diff
@@ -75,9 +75,13 @@ def test_feature_simple_run_once_compute_totals(self):
...
 
-        assert_rel_error(self, prob['comp.f_xy'], -15.0)
+        totals = prob.compute_totals(of=['comp.f_xy'], wrt=['p1.x', 'p2.y'])
+        print("Total derivative of 'comp.f_xy' wrt to 'p1.x' is {}".format(totals[('comp.f_xy','p1.x')][0][0]))
+        print("Total derivative of 'comp.f_xy' wrt to 'p2.y' is {}".format(totals[('comp.f_xy','p2.y')][0][0]))
 
-        prob.compute_totals(of=['comp.f_xy'], wrt=['p1.x', 'p2.y'])
+        totals = prob.compute_totals(of=['comp.f_xy'], wrt=['p1.x', 'p2.y'], return_format= 'dict')
+        print("Total derivative of 'comp.f_xy' wrt to 'p1.x' is {}".format(totals['comp.f_xy']['p1.x'][0][0]))
+        print("Total derivative of 'comp.f_xy' wrt to 'p2.y' is {}".format(totals['comp.f_xy']['p2.y'][0][0]))
```
Comment (swryan) [inline openmdao/core/tests/test_problem.py]: shouldn't these be asserts? (which are converted to prints for the docs)?

Comment (kmarsteller) [reply openmdao/core/tests/test_problem.py]: Good point.  If we are just printing out values, the test will never actually fail?

Comment (kmarsteller) [reply openmdao/core/tests/test_problem.py]: I definitely got confused between what I wanted to see in the docs and what should be in the actual test.

**Comment by hschilling** [comment]
Will change

**Comment by hschilling** [comment]
Fixed and pushed

File: openmdao/docs/features/core_features/working_with_derivatives/index.rst [hunk 2]
```diff
@@ -11,6 +11,8 @@ Working with Derivatives
     sparse_partials.rst
     specifying_partials.rst
     approximating_totals.rst
+    compute_totals.rst
+    compute_totals.rst
```
Comment (swryan) [inline openmdao/docs/features/core_features/working_with_derivatives/index.rst]: duplicate entry

Comment (kmarsteller) [reply openmdao/docs/features/core_features/working_with_derivatives/index.rst]: good catch

**Pull request merged**